UI/HtmlControls/HtmlControls/HtmlOption.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Markup;
+using Uno.UI.Runtime.WebAssembly;
+
+namespace HtmlControls
+{
+    [HtmlElement("option")]
+    [ContentProperty(Name = nameof(Content))]
+    public partial class HtmlOption : UIElement
+    {
+        public static DependencyProperty ContentProperty { get; } = DependencyProperty.Register(
+            "ContentProperty", typeof(string), typeof(HtmlOption), new PropertyMetadata(null, OnContentChanged));
+
+        public string Content
+        {
+            get => (string)GetValue(ContentProperty);
+            set => SetValue(ContentProperty, value);
+        }
+
+        private static void OnContentChanged(DependencyObject o, DependencyPropertyChangedEventArgs args)
+        {
+            if (o is HtmlOption option)
+            {
+                if (args.NewValue is string str)
+                {
+                    // TODO: Clearchildren no longer accessible
```

Comment:
@carldebilly can you suggest what we should do here as ClearChildren isn't accessible as it's internal